README.rst

```diff
@@ -351,6 +352,48 @@ mode (see **Using transparent mode** above) or for a specific request (see
 **Sending requests with automatically-mapped parameters** above), Zyte API
 parameters are chosen as follows by default:
 
+-   ``Request.url`` becomes ``url``, same as in requests with manually-defined
+    parameters.
+
+-   If ``Request.method`` is something other than ``"GET"``, it becomes
+    ``httpRequestMethod``.
+
+-   ``Request.headers`` become ``customHttpRequestHeaders``.
+
+-   ``Request.body`` becomes ``httpRequestBody``.
```

Comment:
Moved at the top to get the short bullet points out of the way earlier and have the more verbose bullet points towards the end.